locale/en_US/default.po

```diff
@@ -140,5 +140,7 @@ msgstr "R"
 msgid "default.contextSettings.emailSignature"
 msgstr ""
 "<br/>\n"
-"________________________________________________________________________<br/>\n"
-"<a href=\"{$contextUrl}\">{$contextName}</a>"
+"<hr /><br/>\n"
+"<p class='emaiSignatureContextUrlStyle'><a href=\"{$contextUrl}\">{$contextName}</a></p>"
+"<p class='emaiSignatureMailingAddressStyle'>{$mailingAddress}</p>"
```

Comment:
I don't think the mailing address is rich text, so I think you would need to use `nl2br` to prevent it from all running onto one line when it's presented in HTML -- and `escape` to properly format special characters.